templates/export/generic-post-type.blade.php

```diff
@@ -16,9 +16,9 @@
 		{!! $append_post_content !!}
 	@endif
 	@if( isset( $endnotes ) )
-		{{ $endnotes }}
+		{!! $endnotes !!}
```

Comment:
We can have footnotes and endnotes with paragraph, images, and other html elements.